File: bindings/go/binding_test.go
package tree_sitter_bitbake_test

import (
	"testing"

	tree_sitter "github.com/tree-sitter/go-tree-sitter"
	tree_sitter_bitbake "github.com/tree-sitter-grammars/tree-sitter-bitbake/bindings/go"
)

func TestCanLoadGrammar(t *testing.T) {
	language := tree_sitter.NewLanguage(tree_sitter_bitbake.Language())
	if language == nil {
		t.Errorf("Error loading BitBake grammar")
	}
}

File: bindings/node/binding_test.js
import assert from "node:assert";
import { test } from "node:test";
import Parser from "tree-sitter";

test("can load grammar", async () => {
  const parser = new Parser();

  await assert.doesNotReject(async () => {
    const { default: language } = await import("./index.js");
    parser.setLanguage(language);
  });
});
